packages/augur-core/source/contracts/Augur.sol

```diff
@@ -229,22 +229,22 @@ contract Augur is IAugur {
     //
 
     // This signature is intended for the categorical market creation. We use two signatures for the same event because of stack depth issues which can be circumvented by maintaining order of paramaters
-    function logMarketCreated(uint256 _endTime, bytes32 _topic, string memory _description, string memory _extraInfo, IMarket _market, address _marketCreator, int256 _minPrice, int256 _maxPrice, IMarket.MarketType _marketType, bytes32[] memory _outcomes) public returns (bool) {
+    function logMarketCreated(uint256 _endTime, bytes32 _topic, string memory _extraInfo, IMarket _market, address _marketCreator, address _designatedReporter, uint256 _feeDivisor, int256[] memory _prices, IMarket.MarketType _marketType, bytes32[] memory _outcomes) public returns (bool) {
         IUniverse _universe = IUniverse(msg.sender);
         require(isKnownUniverse(_universe));
         recordMarketShareTokens(_market);
         markets[address(_market)] = true;
-        emit MarketCreated(_universe, _endTime, _topic, _description, _extraInfo, _market, _marketCreator, _minPrice, _maxPrice, _marketType, 10000, _outcomes);
+        emit MarketCreated(_universe, _endTime, _topic, _extraInfo, _market,_marketCreator, _designatedReporter, _feeDivisor, _prices, _marketType, 10000, _outcomes);
         return true;
     }
 
     // This signature is intended for yesNo and scalar market creation. See function comment above for explanation.
-    function logMarketCreated(uint256 _endTime, bytes32 _topic, string memory _description, string memory _extraInfo, IMarket _market, address _marketCreator,int256 _minPrice, int256 _maxPrice, IMarket.MarketType _marketType, uint256 _numTicks) public returns (bool) {
+    function logMarketCreated(uint256 _endTime, bytes32 _topic, string memory _extraInfo, IMarket _market, address _marketCreator, address _designatedReporter, uint256 _feeDivisor, int256[] memory _prices, IMarket.MarketType _marketType, uint256 _numTicks) public returns (bool) {
```

Comment:
Would these function signatures still work (not produce stack depth errors) if we kept them as _minPrice and _maxPrice?

If so I think that's a bit nicer of a function signature. We only really need to use the array format for the log if we can get away with that.

Reply:
I don't think so since MarketCreated already has 12 params, but I can double-check.

Reply:
Yep, I got a stack-depth error when changing these signatures to accept _minPrice & _maxPrice as separate params. I think we can output them as separate params in the actual MarketCreated log, though. Do you think it would be better to make that change?

Reply:
Oh woops I commented in the wrong function. this was intended to be about `Universe.createScalarMarket`

Reply:
That's the one I changed the signature for, but it still gave the stack-depth error.

Reply:
No luck. :-/ Changing _prices to _minPrice & _maxPrice in createScalarMarket's signature and then passing them into augur.logMarketCreated as _prices still throws the error.